docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+   ```  
+
+   Or, if Traefik is unavailable and your `domain2` Administration Server pod is running, you can run `kubectl exec`:
+
+   ```  
+   $ kubectl exec -n sample-domain1-ns sample-domain2-admin-server -- bash -c \ 
+     "curl -s -S -m 10 http://sample-domain2-cluster-cluster-1:8001/myapp_war/index.jsp"
+   ```  
+
+   You should see something like the following:
+
+   {{%expand "Click here to see the expected web application output." %}}
+
+   ```  
+   <html><body><pre>
+   *****************************************************************
+
+   Hello World! This is version 'v1' of the mii-sample JSP web-app.
+
+   Welcome to WebLogic server 'managed-server1'!
+
+    domain UID  = 'sample-domain2'
+    domain name = 'domain2'
+
+   Found 1 local cluster runtime:
+     Cluster 'cluster-1'
+
+   Found 1 local data source:
+     Datasource 'mynewdatasource': State='Running'
+
+   *****************************************************************
+   </pre></body></html>
+
+   ```  
+   {{% /expand%}}
+
+If you see an error, then consult [Debugging]({{< relref "/userguide/managing-domains/model-in-image/debugging.md" >}}) in the Model in Image user guide.
+
+We will not be using the `sample-domain2` domain again in this sample; if you wish, you can shut it down now by calling `kubectl -n sample-domain1-ns delete domain sample-domain2`.
+
+### Update3 use case
+
+The Update3 use case demonstrates deploying an an updated WebLogic application to the running [Update1 use case](#update1-use-case) domain using an updated Docker image.
+
+In the use case, we will:
+
+ - Create an image `model-in-image:WLS-v2` that is similar to the currently active `model-in-image:WLS-v1` image, but with with the following updates:
+   - An updated web application `v2` at the `myapp-v2` directory path within the WDT application archive instead of `myapp-v1`.
+   - An updated model YAML within the image that points to the new web application path.
+ - Apply an updated domain resource that references the new image while still referencing the original [Update1 use case](#update1-use-case) case secrets and model ConfigMap. 
+
+Once the updated domain resource is applied, we expect the operator to:
+
+ - Rerun the introspector job and generate a new domain home based on the new model.
+ - Restart the domain's administration server pod so that it loads the new image and new domain home.
+ - Roll the domain's cluster servers one at a time so that they each load the new image, new domain home, and revised application.
+
+Finally, we will call the application to verify that its revision is active.
+
+Note that the old version of the application `v1` remains in the new image's archive but is unused. We leave it there to demonstrate that the old version can remain in case you want to revert to it. Once the new image is applied, you can revert by modifying your model's `configuration.model.configMap` to override the related application path in your image model.
```

Comment:
Once the new image -> After the new image

Reply:
done